pyteamcity.py

```diff
@@ -124,18 +161,14 @@ def get_all_builds(self, branch='',
         :param count: how many builds to return
         """
 
-    @GET('builds/'
-         '?locator=buildType:{build_type_id},branch:{branch}'
-         '&start={start}&count={count}')
-    def get_all_builds_by_build_type_id(self, build_type_id, branch='',
-                                        start=0, count=100):
+    @GET('builds/?locator={locator}&start={start}&count={count}')
+    def get_all_builds_locator(self, locator='', start=0, count=100):
```

Comment:
I think this and `get_all_builds` should be hidden actually (prefixed with an underscore), because they are called by `get_builds`, which is an easier to use façade.